storage/storage-proto/src/proto/storage.proto

```diff
@@ -167,8 +167,11 @@ message GetEpochChangeLedgerInfosRequest {
 }
 
 message GetEpochChangeLedgerInfosResponse {
-    /// Vector of latest ledger infos per epoch (not sorted)
+    // Vector of latest ledger infos per epoch (not sorted)
```

Comment:
What does "not sorted" mean here...?

Reply:
it's probably out-dated, it's used to an unsorted Vec<Proof>?